Shapes/shapes/v2/datatype_map.ttl

```diff
@@ -0,0 +1,139 @@
+###############################################################################
+# RML Datatype Map and datatype shortcut shape                                #
+# Copyright Dylan Van Assche, IDLab - UGent - imec (2023)                     #
+###############################################################################
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix schema: <http://schema.org/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rml: <http://w3id.org/rml/core/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> . 
+
+schema:RMLDatatypeMapShape
+    a sh:NodeShape ;
+    sh:ignoredProperties (rdf:type) ;
```

Comment:
why is this property ignored?

Reply:
Should be dropped, it is from `sh:closed` as @pmaria pointed out, will fix :)